packages/field-content/src/Implementation.js

```diff
@@ -224,7 +218,7 @@ export class Content extends Relationship {
   extendAdminViews(views) {
     return {
       ...views,
-      blocks: unique(flatMap(this.blocks, block => block.getAdminViews())),
+      blocks: unique(flatMap(this.blocks, block => block.getAdminViews({ availableBlocks }))),
```

Comment:
Dependency inject so that other blocks can depend-upon the defaults.

Reply:
Why dependency injection over exporting the available blocks?

Reply:
Because they're instances which aren't instantiated until the `Content` constructor is executed.

Then later on, those instances can dynamically define their view requirements (via `.getAdminViews()`).

Why are they instances? Because blocks such as the OEmbed block take an adapter ("iFramely" or "embedly"), and it's up tot he adapter to determine what views to include. That adapter isn't known until execution time.

Reply:
I'm a little confused, `availableBlocks` is importing the blocks, right?
https://github.com/keystonejs/keystone-5/blob/3082dce73ea272f5d7bc8b6fe008d7bbd6e145fe/packages/field-content/src/Implementation.js#L4

And those aren't instances, are they?

Reply:
🤦‍♂ You're right! And the blocks are being instantiated on demand.

I was thinking of the blocks injected into the `Node`/`Sidebar` components exported from the `Block` views.

I'll remove this injection and just export the blocks like you suggested 👍

Reply:
Done 👍